_posts/2025-05-06-jp7914-blogpost.md

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
remove this first empty line - its causing jekyll to break and not show the blog correctly

Reply:
```suggestion
```